### DISCUSSION TITLE
Is there a way to receive POST data when the network type is PING?

### DISCUSSION THREAD

**Discussion by BeomsooIn** [MisterPhilip/omnibug#215] [Q&A] [answered]
I am currently creating a custom provider by downloading the code. There is no problem receiving POST data of the regular fetch type, but when the network type is PING, an empty value is passed to parsePostData(). Is there any way to resolve this?

thanks

**Comment by MisterPhilip** [comment, marked as answer]
Unfortunately, not right now due to an open Chromium bug (see #213) 😢

**Reply by BeomsooIn** [reply]
Thank you for your response. I have verified that the data can be viewed correctly in Firefox instead of Chrome.

I have an additional question. Is it possible to filter by Network Type? (I want to remove "Options" Type)

---------------------

It seems to be in the eventPage.js file. I will check it. 

Thank you for your prompt response, and I'm grateful that you are able to use this useful tool.

The sentence may not be smooth using a translator... Thank you :)

**Reply by MisterPhilip** [reply]
I thought it was already filtering out Options requests - but if not, it would need to be in the eventPage.js file, correct.
